includes/server-timing/hooks.php

```diff
@@ -0,0 +1,37 @@
+<?php
+/**
+ * Hook callbacks used for Server Timing.
+ *
+ * @package performance-lab
+ *
+ * @since n.e.x.t
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+
+/**
+ * Adds server timing to REST API response.
+ *
+ * @since n.e.x.t
+ *
+ * @param WP_REST_Response|WP_Error $response Result to send to the client. Usually a `WP_REST_Response`.
+ * @return WP_REST_Response|WP_Error Filtered response.
+ */
+function rest_post_dispatch_add_server_timing( $response ) {
+	if ( ! defined( 'REST_REQUEST' ) || ! REST_REQUEST || ! function_exists( 'perflab_server_timing' ) || ! $response instanceof WP_REST_Response ) {
+		return $response;
+	}
+
+	$server_timing = perflab_server_timing();
+
+	/** This filter is documented in includes/server-timing/class-perflab-server-timing.php */
+	do_action( 'perflab_server_timing_send_header' ); // phpcs:ignore WordPress.NamingConventions.PrefixAllGlobals.NonPrefixedHooknameFound
```

Comment:
Why is suppressing `WordPress.NamingConventions.PrefixAllGlobals.NonPrefixedHooknameFound` needed here? It's not being triggered by our PHPCS config. I think it should be removed:

```suggestion
	do_action( 'perflab_server_timing_send_header' );
```

Perhaps it's due to a local PHPCS config you have? Perhaps we should add the `WordPress.NamingConventions.PrefixAllGlobals` sniff to this repo and configure it to require the plugin-specific prefix(es).